client/client.go

```diff
@@ -21,21 +21,25 @@ import (
 var (
 	DefaultTimeout time.Duration = time.Second
 	Null                         = byte('\x00')
+	NullBytes                    = []byte{Null}
 )
 
 // One client connect to one server.
 // Use Pool for multi-connections.
```

Comment:
Is this still a valid comment?

Reply:
Yes

Reply:
Ah, the Pool is for multiple servers, even if we now Pool our requests and responses.